writehat/templates/panes/engagementReportsList.html

```diff
@@ -4,27 +4,30 @@
       <h4>Reports: {{ engagement.reports|length }}</h4>
     </div>
     <div class='ml-auto'>
+      {% include 'snippets/statusButtonGroup.html' %}
```

Comment:
Can we add a label to this button group, like `Show only:`, or a tooltip icon indicating that it's meant to filter which reports are displayed?